### ISSUE TITLE
feature request: store the path from which you are uploading files

### ISSUE THREAD

**Issue by JimmyKater** [JuergenFleiss/aTrain#32]
Hey there,

I'm running v 1.1.0 on a win10-machine (if you'f need more infos, pls let me know)

it seems as if you have to set the uploading path anew for each file you want to have transcribed. 
If the app would store the last path from where files were uploaded it would be easier to handle

**Comment by JuergenFleiss** [comment]
yes, not only that, but we really should store all the settings you use. It's at the very top of the list.
